test/quantization/fx/test_quantize_pt2e.py

```diff
@@ -299,6 +314,61 @@ def forward(self, x):
         self.checkGraphModuleNodes(
             m, expected_node_list=node_list, expected_node_occurrence=node_occurrence)
 
+    def test_prepare_qat_conv_bn_fusion(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 3)
```

Comment:
can you also add a test where we have conv + conv + bn?

Reply:
Ok, let's do it in a future PR